index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="styles.css">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Page</title>
+</head>
+<body>
+	
+  <header class="about-me">
+		<div class="menu-bar">
+      <a href="#">my logo</a>
+      <img src="img/Union.svg" alt="Hamburger">
+		</div>
```

Comment:
### OPTIONAL

- [ ] Instead of `div`  you can use `nav` tag. Using HTML semantic tags good for [SEO](https://www.pluralsight.com/guides/semantic-html). 

- [ ] Also you are using 8 indent space. Please use 2 space. Check this [article](https://developers.google.com/style/html-formatting).